frontend/src/metabase-lib/lib/DimensionOptions/DimensionOptions.ts

```diff
@@ -38,8 +38,7 @@ export default class DimensionOptions {
   sections({ extraItems = [] } = {}): DimensionOptionsSection[] {
     const [dimension] = this.dimensions;
     const table = dimension && dimension.field().table;
-    const tableName =
-      table && !table.isSavedQuestion() ? table.objectName() : null;
+    const tableName = table ? table.objectName() : null;
```

Comment:
I know this has side effects for the filter sidebar, and I'm comfortable with those. Not sure what else this might affect though 🤔

Reply:
@daltojohnso , it looks like you have touched https://github.com/metabase/metabase/blob/master/frontend/src/metabase/parameters/utils/mapping-options.js before, which appears to be one of the few places this code is called. Can you think of why we would *not* want to show a table name when the table is a saved question?

Reply:
I'm not entirely sure, but the change leads back to PR https://github.com/metabase/metabase/pull/13427 which addressed both https://github.com/metabase/metabase/issues/13376 and https://github.com/metabase/metabase/issues/13406. I think @mazameli could help provide more context on that one.

I tried to test what #13427 does and it seems this change doesn't break it so I guess (with my limited knowledge) it should be fine.
![image](https://user-images.githubusercontent.com/1937582/173774577-bfdebc49-6c17-4ebb-810d-400c6cbab4c0.png)